app.rb

```diff
@@ -21,23 +21,23 @@ def fixture(path)
 
       case action
       when 'http://schemas.xmlsoap.org/ws/2005/02/trust/RST/SCT'
-        sleep ENV['AAMVA_SECURITY_TOKEN_DELAY'].to_i
+        sleep ENV['AAMVA_SECURITY_TOKEN_DELAY'].to_f
```

Comment:
Does the kernel method also round? https://ruby-doc.org/core-2.7.2/Kernel.html#method-i-sleep

> Suspends the current thread for duration seconds (which may be any number, including a Float with fractional seconds). Returns the actual number of seconds slept (rounded), which may be less than that asked for if another thread calls Thread#run.

I can't tell if that means that it rounds the output value or the actual time slept.

Reply:
yay ruby, LGTM

```
% time ruby -e '100.times { sleep(0.49) }'
ruby -e '100.times { sleep(0.49) }'  0.05s user 0.01s system 0% cpu 49.268 total
```